tf-modules/archive/db_indexer.tf

```diff
@@ -25,7 +25,6 @@ resource "aws_lambda_function" "db_indexer" {
       CollectionsTable           = var.dynamo_tables.collections.name
       ExecutionsTable            = var.dynamo_tables.executions.name
       AsyncOperationsTable       = var.dynamo_tables.async_operations.name
-      FilesTable                 = var.dynamo_tables.files.name
```

Comment:
Are the variables for granules table and executions table still necessary?

Reply:
Good question - they may not be.    I'm adding tickets to address table removal and/or model removal, perhaps this should go there?